tests/test_workers.py

```diff
@@ -32,6 +32,8 @@ async def test_sentry(amqp_connection, config, nursery):
     assert client.dsn == sentry_dsn
     version = pkg_resources.get_distribution("vumi2").version
     assert client.options["release"] == version
+    # Disable sentry for the rest of the tests
+    sentry_sdk.init()
```

Comment:
Should this be in a `try`/`finally` block (or an equivalent fixture)?

Also, since we never `await` in here, should the test be sync instead of async?

Reply:
So because the fixtures are async, the tests needs to be async to be able to use them, otherwise pytest gives an error, so that's the reason why all tests are async even when they don't need to be.

A fixture doesn't seem to be the correct fit for this (since sentry is configured globally, you can't pass it down to the worker class), but it should be wrapped in a `try`/`finally` block, so I'll do that.